draft-ietf-mls-protocol.md

```diff
@@ -2006,6 +2006,14 @@ The Commit MUST NOT combine proposals sent within different epochs. In the event
 that a valid proposal is omitted from the next Commit, the sender of the
 proposal SHOULD retransmit it in the new epoch.
 
+A member of the group MAY send a Commit that references no proposals at all,
+which would thus have empty vectors for `updates`, `removes`, and `adds`.  Such
+a Commit resets the sender's leaf and the nodes along its direct path and
```

Comment:
I don't believe such a commit would change the leaf. Makes more sense to me to not allow empty commits

Reply:
It would reset the leaf, via `Commit.path.leaf_key_package`.

ISTM there is a very concrete use case for this feature, namely keeping the tree full.  If you want to maintain a full tree, then you would want to have each new joiner should send an empty commit immediately on joining, have the neighbor of a removed node send an empty commit after a remove, etc.